database/mongoDB/initMongo.sh

```diff
@@ -159,12 +159,12 @@ mongosh qa \
 
 # ensure photo_ is unique
 # mongosh qa \
-#   --eval 'db.answersPhotos.createIndex( { photo_id: 1 }, { unique: true } )'
+#   --eval 'db.answersPhotos.createIndex( { id: 1 }, { unique: true } )'
 
 # test insert
 # mongosh qa \
 #   --eval 'db.answersPhotos.insertOne({
-#     photo_id: 4,
+#     id: 4,
 #     answer_id: 2,
 #     url: "https://images.unsplash.com/photo-1500603720222-eb7a1f997356?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1653&q=80"
 #   })'
```

Comment:
I think this is because I'm unfamiliar with sh files but does the "#" mean all of these lines are commented out? I